seed/seed16.go

```diff
@@ -350,10 +377,17 @@ func (s *seed16) LoadEssentialMeta(essentialTypes []snap.Type, tm timings.Measur
 
 	s.resetSnaps()
 
-	return s.loadEssentialMeta(essentialTypes, required, added, tm)
+	if handler == nil {
+		handler = defaultSnapHandler{}
+	}
+
+	if len(essentialTypes) == 0 {
```

Comment:
this is a slightly unrelated change because I caught that we had changed the seed20 behavior about this but not the seed16, see also related test change below